### DISCUSSION TITLE
ZMK - How to lock game layer

### DISCUSSION THREAD

**Discussion by fidrox** [manna-harbour/miryoku#236] [Q&A] [answered]
board: nice_nano_v2
shield: corne_left,corne_right
base: QWERTY
pcb: chocofi
custom_config.h: [link](https://github.com/fidrox/miryoku_zmk/blob/master/miryoku/custom_config.h)

Sorry if this has been asked endless times but I could not find and example. Seems like there are some limitations with zmk when it comes to tg layers, but is there a way I can toggle a game layer on? 

Saw opp mentioned in the docs but when testing it with the NUM layer it did not do anything (hold left middle thumb button and use left middle finger on opp). Looking through the config files I only see U_LT and U_MT as options.

Someone was able to set U_GAME to an unused key, but keys are limited on this board.

**Comment by manna-harbour** [comment, marked as answer]
[Using opposite layer lock as an alternative to current layer lock](https://github.com/manna-harbour/miryoku_zmk#no-current-layer-lock) only applies to the half-layers activated via layer-tap (Num, Nav, etc.).

For a Game layer you can [substitute](https://github.com/manna-harbour/miryoku/discussions/85) the [included Extra or Tap layers](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#additional-features). Tap may even be usable for games as-is.

In all cases [you need to double tap](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#additional-features).

**Comment by fidrox** [comment]
Thank you so much for the reply, just figured out the tap function and that fits me needs perfectly.
